src/components/SearchResultCard.tsx

```diff
@@ -0,0 +1,108 @@
+import Card from '@material-ui/core/Card';
+import CardActions from '@material-ui/core/CardActions';
+import CardContent from '@material-ui/core/CardContent';
+import React, { useState } from 'react';
+import makeStyles from '@material-ui/core/styles/makeStyles';
+import { Grid, Typography } from '@material-ui/core';
+import { getRecipeById } from '../services';
+import { RecipeById } from '../models';
+import { styled } from '@mui/material/styles';
+import Collapse from '@mui/material/Collapse';
+import IconButton, { IconButtonProps } from '@mui/material/IconButton';
+import ExpandMoreIcon from '@mui/icons-material/ExpandMore';
+import ReactHtmlParser from 'react-html-parser'; 
+
+
+export interface SearchResultCardProperties {
+    title: string | undefined;
+    description: number;
+    image: string | undefined;
+    actions?: React.ReactNode[];
+  }
+
+  const useStyles = makeStyles(({palette}) => ({
+    card: {
+      height: '100%',
+      //display: 'flex',
+      //flexWrap: 'wrap',
```

Comment:
best practice is to remove comments or leave a `TODO` comment if u plan to come back to something - but its best to just create an issue to track it instead of leaving todo comments.